lib/bumblebee/text/text_classification.ex

```diff
@@ -6,10 +6,13 @@ defmodule Bumblebee.Text.TextClassification do
   def text_classification(model_info, tokenizer, opts \\ []) do
     %{model: model, params: params, spec: spec} = model_info
     Shared.validate_architecture!(spec, :for_sequence_classification)
-    opts = Keyword.validate!(opts, [:compile, top_k: 5, defn_options: []])
+
+    opts =
+      Keyword.validate!(opts, [:compile, top_k: 5, scores_function: "softmax", defn_options: []])
```

Comment:
```suggestion
      Keyword.validate!(opts, [:compile, top_k: 5, scores_function: :softmax, defn_options: []])
```

Same for the rest :)

Reply:
And let's also document the new option in [respective places](https://github.com/elixir-nx/bumblebee/blob/776e57c6b6d06c0fed47afa26d8144c7c2541149/lib/bumblebee/text.ex#L257).

Reply:
Thanks! Adjusted as per your suggestions.